app/assets/stylesheets/shelf_items.scss

```diff
@@ -0,0 +1,131 @@
+@import "variables";
+
+.shelf-items {
+  .banner.banner-primary-1 .banner-content {
+    padding: 3rem 2rem 2rem;
+
+    h4 {
+      text-align: center;
+      color: white;
+    }
+  }
+
+  .shelf {
+    flex-wrap: wrap;
+    justify-content: center;
+    height: 100%;
+
+    .shelf-item .brew-link {
+      text-align: center;
+      margin-top: 1rem;
+
+      a {
+        display: inline-block;
+        font-weight: 600;
+        font-size: 1.2rem;
+        padding: 0.5rem 1.5rem;
+        border-radius: 2rem;
+        border: 1px solid $color-primary-2;
+        color: $color-primary-2;
+        transition: background-color 0.25s, color 0.25s, border-color 0.25s;
+
+        &:hover, &:focus {
+          background-color: $color-primary-opacity-10;
+          border-color: $color-primary-opacity-10;
+          color: $color-primary-0;
+        }
+      }
```

Comment:
Can we use the classes `link link-primary` on the a tags

Reply:
There was some reason why this wouldn't work I think, but I'll give it a go. Definitely will make it DRYer

Reply:
Can't change this because this is conditional. This style only applies on the index page not the dashboard page. Both pages use the same HTML but different styling.